### DISCUSSION TITLE
Fuzzy Searching Collections

### DISCUSSION THREAD

**Discussion by andylonsdale** [payloadcms/payload#6766] [Q&A]
How would i go about implementing Fuzzy search for a collections search bar?

By default payload is using full text search but it would be good to have the ability to fuzzy find on larger collections

**Comment by matthijs166** [comment]
I think that the underlying database needs to support it, it's a quite heavy task if it is not optimized.

Some good options are https://www.algolia.com/ or https://typesense.org/

Downsides is that you need to integrate it.
